props/saber_caiwyn_buttons.h

```diff
@@ -338,92 +311,76 @@ class CaiwynButtons : public PropBase {
     track_player_ = GetFreeWavPlayer();
```

Comment:
Since NextTrack doesn't free the track player anymore, there is a now a minor problem here.
GetFreeWavPlayer() can fail to allocate a player even if we already have one, because it's reserved and already stored in track_player_.

I recommend:

if (!track_player_) track_player_ = GetFreeWavPlayer();

Reply:
I assumed PropBase::Loop() would resolve that by freeing the track player after it was stopped.  Is that not the case?

Reply:
This is done.